rialto_airflow/dags/harvest.py

```diff
@@ -176,9 +182,13 @@ def link_funders(snapshot, openalex_fill_in, dimensions_fill_in):
         snapshot, openalex_jsonl, dimensions_jsonl, wos_jsonl
     )
 
-    distill_publications(snapshot, openalex_fill_in, dimensions_fill_in)
+    distilled_pubs = distill_publications(
+        snapshot, openalex_fill_in, dimensions_fill_in
+    )
+
+    linked_pubs = link_funders(snapshot, openalex_fill_in, dimensions_fill_in)
 
-    link_funders(snapshot, openalex_fill_in, dimensions_fill_in)
+    (distilled_pubs, linked_pubs) >> publish_openaccess(snapshot)
```

Comment:
Once the DAG settles a bit I'd like to rewire up the tasks using `>>` where possible, so we don't have to pass around unneeded dependencies?